R/downloadableReactTable.R

```diff
@@ -0,0 +1,229 @@
+# -------------------------------------------
```

Comment:
expand the top/bottom to match and end with the three dashes at the end of the middle line.

Reply:
done